kernel/src/vtpm/ptp/crb.rs

```diff
@@ -247,6 +255,41 @@ fn locality_from_addr(addr: u32) -> u8 {
     (addr >> 12) as u8
 }
 
+fn hash_sha384(hash_data: &str, digest_sha384: &mut [u8; TPM2_SHA384_SIZE]) -> Result<()> {
```

Comment:
These functions can be moved to tdx folder, e.g., add a `measurement.rs`

Reply:
moved to tdx folder.